MapboxCoreNavigation/RouteController.swift

```diff
@@ -139,6 +142,11 @@ open class RouteController: NSObject {
         super.init()
         
         self.locationManager.delegate = self
+        
+        // This is just a test!!! (needs a real access token, a user agent base, and the actual nav SDK version
+        events.initialize(withAccessToken: "your token here!", userAgentBase: "MapboxEventsiOSNavigation", hostSDKVersion: "42.42.42")
+        events.isDebugLoggingEnabled = true
+        events.sendTurnstileEvent()
```

Comment:
I think @boundsj might just be showing an example of how you would use the API here?